redi-connect-backend/scripts/20190611_second-data-import.js

```diff
@@ -1,9 +1,9 @@
-'use strict';
+"use strict";
```

Comment:
They changed actually because I changed the links in the email... so the formatter kicked in. 
Anyway the emails will be touched later anyway so Im happy to remove those changes.

Reply:
👌